pkg/config.go

```diff
@@ -804,8 +804,13 @@ func LoadConfig(configFiles []string, deploymentId int) (*Config, error) {
 					Methods:           ParseHttpMethods([]string{"POST"}),
 					SetRequestHeaders: headers,
 				},
+				// checkout repo
+				AllowlistItem{
+					URL:               bitBucketBaseUrl.JoinPath("/scm/:project/:repo.git/*").String(),
```

Comment:
```suggestion
					URL:               bitBucketBaseUrl.JoinPath("/scm/:project/:repo/*").String(),
```
nit: `:repo.git` will match everything between slashes, so we might want to remove the `.git` from that just to prevent confusion (i.e. it will match things that dont end in .git too)

Reply:
Ah good call. Updated in latest commit